extras/docker/Dockerfile

```diff
@@ -1,92 +1,21 @@
-# w3af.org
-# https://github.com/andresriancho/w3af/tree/master/extras
+FROM ubuntu:14.04
 
-FROM ubuntu:12.04
-MAINTAINER Andres Riancho <andres.riancho@gmail.com>
```

Comment:
I didn't, although it looks like it.

I built this Dockerfile from scratch instead of changing the existent, so the differences started to appear. The same applies for the lines above ("# w3af.org",etc )

Will add those lines, don't worry.